libraryversions.toml

```diff
@@ -235,7 +235,7 @@ KRUTH = { group = "androidx.kruth", atomicGroupVersion = "versions.KRUTH" }
 LEANBACK = { group = "androidx.leanback" }
 LEGACY = { group = "androidx.legacy" }
 LIBYUV = { group = "libyuv", atomicGroupVersion = "versions.LIBYUV" }
-LIFECYCLE = { group = "androidx.lifecycle", atomicGroupVersion = "versions.LIFECYCLE" }
+LIFECYCLE = { group = "org.jetbrains.androidx.lifecycle", atomicGroupVersion = "versions.LIFECYCLE" , overrideInclude = [ ":lifecycle:lifecycle-common", ":lifecycle:lifecycle-runtime", ":lifecycle:lifecycle-runtime-compose" ]}
```

Comment:
Do we need `overrideInclude`?

Reply:
for some reason, without `overrideInclude` the publication goes to the original group - androidx.lifecycle
I'm not sure why.